docs/modules/ROOT/pages/licensing.adoc

```diff
@@ -0,0 +1,106 @@
+* Jaeger: Apache 2.0 https://github.com/jaegertracing/jaeger/blob/master/LICENSE
+* Nginx: https://github.com/nginxinc/docker-nginx/blob/master/LICENSE
+* Prometheus Merge Tool: Apache 2.0 https://github.com/lablabs/prometheus-alert-overrider/blob/master/LICENSE
+* Alpine.js: MIT https://github.com/alpinejs/alpine/blob/main/LICENSE.md
+* Couchbase Cluster Monitor: Proprietary to Couchbase https://github.com/couchbaselabs/cbmultimanager/blob/master/LICENSE
+
+All licences are in the source repository and the microlith container in the link:https://github.com/couchbaselabs/observability/blob/main/microlith/licenses/[`/licenses`] directory.
+
+A statement is printed out to standard output/console at start up to indicate acceptance of licensing and where you can find them all.
+
+A simple link:https://github.com/couchbaselabs/observability/blob/main/tools/build-oss-container.sh[helper script] is provided as well to build without any proprietary components.
+
+== Container layers
+
+The various external components are reused directly from the layers already published by the entities responsible (e.g. Grafana publish the Loki image layers): this container is not redistributing or repackaging them but using as-is.
+
+The container layers used in the deliverable image are:
+* Grafana: AGPL 3.0 https://github.com/grafana/grafana/blob/main/LICENSE
+* Loki: AGPL 3.0 https://github.com/grafana/loki/blob/main/LICENSE
+* Prometheus: Apache 2.0 https://github.com/prometheus/prometheus/blob/main/LICENSE
+* Alert Manager: Apache 2.0 https://github.com/prometheus/alertmanager/blob/master/LICENSE
+* Jaeger: Apache 2.0 https://github.com/jaegertracing/jaeger/blob/master/LICENSE
+* Nginx: https://github.com/nginxinc/docker-nginx/blob/master/LICENSE
+
+Nginx is used as the base image for the microlith container.
+
+These can all be seen in this excerpt from the link:https://github.com/couchbaselabs/observability/blob/main/microlith/Dockerfile[Dockerfile]:
+
+[source,Dockerfile]
+----
+FROM grafana/grafana:8.1.1 as grafana-official
+FROM grafana/loki:2.3.0 as loki-official
+FROM prom/prometheus:v2.28.1 as prometheus-official
+FROM prom/alertmanager:v0.22.2 as alertmanager-official
+FROM jaegertracing/all-in-one:1.25.0 as jaegaer-official
+
+FROM nginx:alpine
+
+COPY --from=grafana-official /usr/share/grafana /usr/share/grafana
+COPY --from=grafana-official /etc/grafana /etc/grafana
+
+COPY --from=loki-official /usr/bin/loki /usr/bin/loki
+COPY --from=loki-official /etc/loki /etc/loki
+
+COPY --from=prometheus-official /bin/prometheus /bin/prometheus
+COPY --from=prometheus-official /bin/promtool /bin/promtool
+COPY --from=prometheus-official /etc/prometheus/prometheus.yml /etc/prometheus/prometheus.yml
+COPY --from=prometheus-official /usr/share/prometheus/console_libraries/ /usr/share/prometheus/console_libraries/
+COPY --from=prometheus-official /usr/share/prometheus/consoles/ /usr/share/prometheus/consoles/
+----
```

Comment:
Have you intentionally omitted the COPY instructions from Alertmanager and Jaeger?